src/components/init-modal/style.scss

```diff
@@ -294,53 +359,6 @@
 	}
 }
 
-// Buttons group
-.newspack-newsletters-buttons-group {
-	display: flex;
-	justify-content: space-around;
-	&--spaced {
```

Comment:
Getting rid of these styles causes the buttons in the Layout sidebar to look smushed together:

<img width="268" alt="Screen Shot 2022-02-01 at 4 44 45 PM" src="https://user-images.githubusercontent.com/2230142/152070555-b2a8c1c8-2d2d-4000-9450-94e019232689.png">

Reply:
Fixed: https://github.com/Automattic/newspack-newsletters/pull/710/commits/2a5476a70161c0e8c74abccd241321f12cfee3b1